config/kubernetes/sig-docs/teams.yaml

```diff
@@ -1,4 +1,11 @@
 teams:
+  sig-docs-blog-owners:
+    description: Approvers for blog content
+    members:
+    - mrbobbytables
```

Comment:
can you move bob to the maintainers list instead of members?

Reply:
I also added bob to website maintainers in the last squashed commit
or add a maintainer list under `sig-docs-blog-owners`?

Reply:
I ended up moving bob from `sig-docs-blog-owners` to `website-maintainers` because of this error

>      config_test.go:272: The team website-maintainers in org Kubernetes has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: mrbobbytables
>     config_test.go:272: The team sig-docs-blog-owners in org Kubernetes has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: mrbobbytables 

Hopefully I understood what was meant from moving bob to maintainers 😅

Reply:
Still error'd with
>     config_test.go:272: The team website-maintainers in org Kubernetes has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: mrbobbytables

so I added a `maintainers` list 🤞